core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala

```diff
@@ -218,12 +218,6 @@ trait SubmissionMonitor extends FutureSupport with LazyLogging with RawlsInstrum
         workspaceRec <- dataAccess.workspaceQuery.findByIdQuery(submissionRec.workspaceId).result.map(_.head)
       } yield (RawlsUserEmail(submissionRec.submitterEmail), workspaceRec)
 
-    def getPetSAUserInfo(googleProjectId: GoogleProjectId, submitterEmail: RawlsUserEmail): Future[UserInfo] =
```

Comment:
Changed to use existing code in `AutUtil`.

Reply:
I like opportunistic cleanup 😀